test/small-integration-test.sh

```diff
@@ -52,6 +52,7 @@ if [ -n "${ALL_TESTS}" ]; then
         sigv4
         singlepart_copy_limit=10  # limit size to exercise multipart code paths
         #use_sse  # TODO: S3Proxy does not support SSE
+        "use_cache=${CACHE_DIR} -o ensure_diskfree=${ENSURE_DISKFREE_SIZE} -o fake_diskfree=${FAKE_FREE_DISK_SIZE} -o streamupload"
```

Comment:
See the comments below for more details on this point as well.
Since there are no external specification changes, I think we can check the operation without adding tests.
However, it may be possible to add inspections such as testing the boundaries of the upload part.
If you want to add it, it is also necessary for the existing code, so let's deal with it with another issue or PR.

Reply:
I marked Resolved for this review, but if there is a problem, please point it out again to the corrected code or post comments.